apps/api/src/app/events/services/sms-service/handlers/bulksms.handler.ts

```diff
@@ -0,0 +1,25 @@
+import { ChannelTypeEnum } from '@novu/shared';
+import { ICredentials } from '@novu/dal';
+import { BulkSmsSmsProvider } from 'D:/Novu_latest/novu/providers/bulkSms/src/lib/bulkSms.provider';
```

Comment:
```suggestion
import { BulkSmsSmsProvider } from '@novu/bulkSms';
```

I think we can use the package you created here.

Reply:
Yes I have tried the package @novu/bulksms but I think that this package is not present in the node module that's why it's showing error to me. So for that reason I have given the path due to which it can read the file and the function that I have mentioned there. But yes if you have managed it then we can use this @novu/bulksms package here

Reply:
This hasn't been addressed yet.

Reply:
![image](https://user-images.githubusercontent.com/94128805/208412840-4f673183-1b6f-4a2b-b6a1-1195e5055146.png)
@p-fernandez I have also changed this and pushed but still if it is not showing please inform me